cloudformation_templates/aws_jenkins.json

```diff
@@ -122,13 +126,24 @@
       }
     },
 
-    "Route53RecordSet": {
+    "CIRoute53RecordSet": {
+      "Type": "AWS::Route53::RecordSet",
+      "Properties": {
+        "HostedZoneName": {"Ref": "HostedZoneName"},
+        "Name": {"Fn::Join": [".", [{"Ref": "Subdomain"}, {"Ref": "HostedZoneName"}]]},
+        "Type": "A",
+        "ResourceRecords": [
+          {"Ref": "ElasticIP"}
+        ],
+        "TTL": "300"
+      }
+    },
+
+    "LogsRoute53RecordSet": {
       "Type": "AWS::Route53::RecordSet",
       "Properties": {
         "HostedZoneName": {"Ref": "HostedZoneName"},
-        "Name": {"Fn::Join": ["",[
-          {"Ref": "Subdomain"}, ".", {"Ref": "RootDomain"}, "."
-         ]]},
+        "Name": {"Fn::Join": [".", ["*", {"Ref": "LogsSubdomain"}, {"Ref": "HostedZoneName"}]]},
```

Comment:
The commit message says "*.logs.beta domains" - is `stacks.route53zone.outputs.HostedZoneName` always going to evaluate to "beta"?